llvm/lib/Target/PowerPC/PPC.h

```diff
@@ -102,79 +102,99 @@ class ModulePass;
     // PPC Specific MachineOperand flags.
     MO_NO_FLAG,
 
+    /// On PPC, the 12 bits are not enough for all target operand flags.
+    /// Treat all PPC target flags as direct flags. This also means we can not
+    /// use a bitmask flag, so if one operand has two or more flags, a fake
+    /// combination flag must be created. See example MO_GOT_TPREL_PCREL_FLAG.
+
```

Comment:
```cpp
    /// On PPC, 12 bits are not enough as orthogonal masks. Define flags in
    /// sequence to save space. To define new flag, add new enum entry instead
    /// of combining existing flags. See MO_GOT_TPREL_PCREL_FLAG as example.
```

Reply:
For `orthogonal masks`, do you mean bitmask flag? Actually 12 bits are for both "bitmask" flag and "direct flag". Before the change, we have 8 orthogonal mask and 4 bit for direct mask.

Reply:
Yes, I did not catch meaning of _direct flag_ when I wrote the comment. Is `fake combination flag` still confusing though? Maybe `... a new direct flag must be created ...`.